templates/themes/app/content/link_embed.html.twig

```diff
@@ -1,11 +1,14 @@
-{{ ng_view_content(
-    content,
-    'payload',
-    {
-        params: {
+{% if location is defined %}
+    {{ ng_view_content(
+        content,
```

Comment:
`ng_view_content` will accept both `Content` and `Location` objects, and since we are depending on `location` here, it would make more sense to pass it instead of `content`.

Also, please add to the top of the template:

```twig
{# content \Netgen\IbexaSiteApi\API\Values\Content #}
{# location \Netgen\IbexaSiteApi\API\Values\Location #}
```

so we get proper type hinting, every view template should have that.

Same on other places as well.

Reply:
fixed in cac2fd26337f7d37192c4b1aa90a14083b587ee5